hash/pc98.xml

```diff
@@ -69770,6 +69770,118 @@ Same as Police Quest 2 - Quest for Glory stand-alone disks
+    <!-- Roman-Tech Ltd. made the port and Acclaim Japan published it for the PC98 on December 17, 1991 -->
+    <!-- The game runs fine in PC-9801, but the emulations is too fast in PC9821 -->
+    <software name="migrain" supported="yes">
+		<description>Migrain</description>
+		<year>1991</year>
+		<publisher>アクイレムジャパン (Acclaim Japan)</publisher>
+		<info name="alt_title" value="ミグレイン" />
+        <info name="release" value="19911217" />
+        <info name="usage" value="Run MIGRAIN.EXE from DOS" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="1261568">
+				<rom name="Migrain.hdm" size="1261568" crc="c85fa8bb" sha1="149500ffd50734c6f39b93392e9695b8a42e8a35" offset="0" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="momoirom" supported="yes">
+		<description>Momoiro Mahjong</description>
+		<year>1995</year>
+		<publisher>R-Rate</publisher>
+		<info name="alt_title" value="桃色麻雀" />
+		<info name="release" value="1995xxxx" />
+		<part name="flop1" interface="floppy_5_25">
+			<feature name="part_id" value="System Disk"/>
+			<dataarea name="flop" size="1265664">
+				<rom name="Momoiro Mahjong - System Disk.fdi" size="1265664" crc="a93afa80" sha1="0923c8ba47749b68d5e3c2aeea8e252584c202ca" offset="0" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_5_25">
+			<feature name="part_id" value="Disk A"/>
+			<dataarea name="flop" size="1265664">
+				<rom name="Momoiro Mahjong - Disk A.fdi" size="1265664" crc="7b9d4e68" sha1="cf18ea209392363cc5781ed2591e02e38b7734d5" offset="0" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_5_25">
+			<feature name="part_id" value="Disk B"/>
+			<dataarea name="flop" size="1265664">
+				<rom name="Momoiro Mahjong - Disk B.fdi" size="1265664" crc="10268036" sha1="2b29422dce2def3bffdd72bf04fe5a132312d734" offset="0" />
+			</dataarea>
+		</part> 
+	</software>
+
+    <!-- The company Kao used to add a bonus disk in Kao's 10-pack of floppy disks with trial versions of -->
+    <!-- the game and graphics collections as an extra. Eventually, they made the full version of the game -->
+    <!-- available in the bonus disk. This game was not available for sale in regular computer game stores. -->
+    <software name="route246" supported="yes">
+		<description>Kao Floppy Gekijou - Route 246 Satsujin Annai</description>
+		<year>1991</year>
+		<publisher>フェアリーテール (Fairytale)</publisher>
+		<info name="alt_title" value="花王フロッピー劇場 ルート246殺人案内" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="1423360">
+				<rom name="ROUTE_246.fdi" size="1423360" crc="599a082e" sha1="3bd50e646c8f22efc88fa74e00b7ed6521f3d8df" offset="0" />
+			</dataarea>
+		</part>
+	</software>
+
+    <!-- There is no sound when run on PC-9821 -->
+    <!-- PC-9801 has sound but every now and then the game crashes -->
+    <software name="3takun" supported="partial">
```

Comment:
Can you please be more specific on this? 
It looks as a `supported=“no”` from the description given the runs not ensuring workability, from the year I guess it needs an RS class machine (pc9801rs).

PR is fine for the rest.

Reply:
I was not sure if I should set support as “partial” or “no” because the title works on 9821, except for the lack of sound. That’s what lead me to set support as “partial”.  I can change it to “No” if that’s a more accurate statement.